src/consensus/params.h

```diff
@@ -31,6 +67,16 @@ struct BIP9Deployment {
     int64_t nStartTime;
     /** Timeout/expiry MedianTime for the deployment attempt. */
     int64_t nTimeout;
+    // bip135 begin added parameters
```

Comment:
That is a sensible renaming which I'd be happy to make.
I will suggest this in the Pull Request to Core too. It would be nice to keep the common name for the data structure.